Snake/Snake/Style.cs

```diff
@@ -55,62 +44,18 @@ public string StyleMazeElement(int mazeElement)
                     return "▓";
 
                 case 4:
-                    return "■";
+                    return "☻";
```

Comment:
i am sure this is ok for this game but be aware the these unicode characters probably depend on the console also using UTF16 (which it probably does by default or else codepage 850). but if someone happened to change their default font to some other encoding then you might just get jibberish.